src/main/java/org/jenkinsci/plugins/verifystatus/VerificationsPublisher.java

```diff
@@ -173,11 +173,19 @@ public boolean perform(AbstractBuild build, Launcher launcher,
             gerritConfig.getGerritHttpUserName(),
             gerritConfig.getGerritHttpPassword());
     GerritApi gerritApi = gerritRestApiFactory.create(authData);
+    int changeNumber;
+    int patchSetNumber;
+    try {
+      changeNumber = Integer.parseInt(
```

Comment:
Triggerering via gerrit trigger query will actually give the fullset  of vars. 
typically I setup the jobs  as parameterized with a mininal set of necessary params (typcially project, branch, refspec) to  allow testing without voting while keeping  the job in production.